### DISCUSSION TITLE
Component proposal: Quote with attribution

### DISCUSSION THREAD

**Discussion by Courey** [uswds/uswds#5766] [Proposals]
### Component description

I would like to use a component for a quote with attribution. Typically this would be a figure element with a blockquote and a figcaption.

### Describe the solution you'd like

I would like to be able to use a component for a quote with attribution that is consistent and accessible and in line with the USWDS standards. I would like some kind of figure/blockquote/figcaption based component.

### Describe alternatives you've considered

Currently I am using custom styles for this, but I want consistency with USWDS and other government websites.

### Additional context

This is the requested style for my quote component. So I would need some way to add a border, a quote, an attribution with an image. 

<img width="1033" alt="Screenshot 2024-01-02 at 10 12 29 AM" src="https://github.com/uswds/uswds/assets/6545874/2b543ca8-7221-4fa2-884c-e207499a4725">

**Comment by mgifford** [comment]
Would that be similar to https://designsystem.digital.gov/components/summary-box/ but with styled quotes? 

Are there examples you've found you have seen as useful? A couple of from a quick search:

- https://designsystem.utah.gov/library/components/textLayout/blockQuote
- https://cedar.rei.com/components/quote
- https://folwell.umn.edu/typography/pull-quotes-and-block-quotes

**Comment by mejiaj** [comment]
@bonnieAcameron when you have time, could you add the use case for DigitalGov?
